### PR TITLE
Add table captions and anchors

### PR DESCRIPTION
**Pull request by markkramerus**
New formatting for table captions

### CODE REVIEW

**Review by ngfreiter — COMMENTED** [review]
Section 3.6.9 Flag Rules table is not captioned, and section 3.7 Appendix is not captioned. Not sure if those were intentional.

**Comment by markkramerus** [comment]
> Section 3.6.9 Flag Rules table is not captioned, and section 3.7 Appendix is not captioned. Not sure if those were intentional.

I wish .md had soft indexing!

File: input/pagecontent/reference.md [hunk 1]
```diff
@@ -2460,6 +2484,8 @@ Exclude rules are used to remove codes from value sets. Exclude rules appear onl
 
 Flags are a set of information about the element that impacts how implementers handle them. The [flags defined in FHIR](http://hl7.org/fhir/R4/formats.html#table), and the symbols used to describe them, are as follows:
 
+<span class="caption" id="t1">Table 14. Flags and their meaning</span>
```
Comment (ngfreiter) [inline input/pagecontent/reference.md:2487]:
```suggestion
<span class="caption" id="t1">Table 13. Flags and their meaning</span>
```
I know it's unlucky, but I think this is only the 13th table.

**Review by ngfreiter — CHANGES_REQUESTED** [review]

**Comment by markkramerus** [comment]
Oops~

**Review by ngfreiter — APPROVED** [review]
Looks good, thanks!

**Pull request merged**